Editor/UberLoggerEditorWindow.cs

```diff
@@ -365,15 +372,19 @@ bool ShouldShowLog(System.Text.RegularExpressions.Regex regex, LogInfo log)
     /// <summary>
     /// Converts a given log element into a piece of gui content to be displayed
     /// </summary>
-    GUIContent GetLogLineGUIContent(UberLogger.LogInfo log, bool showTimes)
+    GUIContent GetLogLineGUIContent(UberLogger.LogInfo log, bool showTimes, bool showChannels)
     {
         var showMessage = log.Message;
         //Make all messages single line
         showMessage = showMessage.Replace(UberLogger.Logger.UnityInternalNewLine, " ");
-        if(showTimes)
-        {
-            showMessage = log.GetRelativeTimeStampAsString() + ": " + showMessage; 
-        }
+
+        showMessage = string.Format("{0}{1}{2}{3}{4}",
+                showChannels ? "[" + log.Channel + "]" : "",
+                showTimes && showChannels ? " " : "",
+                showTimes ? log.GetRelativeTimeStampAsString() : "",
+                showChannels || showTimes ? " : " : "",
+                showMessage
+            );
```

Comment:
I'm finding the conditional operators hard to parse here. Could we refactor into something like:
var channelMessage = showChannels ? String.Format("[{0}] : ", log.Channel : "";
var timeMessage = showTime ? log.GetRelativeTimeStampAsString() + " : " : "";
showMessage = String.Format("{0}{1}{2}". channelMessage, timeMessage, showMessage);